From: nabijaczleweli <nabijaczleweli@gmail.com>
Date: Sat, 4 Jan 2020 07:24:20 +0100
Subject: Build handler examples with debug symbols

They get stripped later on anyway and it stops dh complaining that the
*-dbgsym package doesn't have any symbols in it
---
 configMakefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configMakefile b/configMakefile
index 4cb9eeb..5e4ac34 100644
--- a/configMakefile
+++ b/configMakefile
@@ -45,7 +45,7 @@ else
 endif
 
 OBJ := .o
-CXXAR := -O3 -std=c++14 -pedantic -Wall -Wextra -pipe $(INCCXXAR) $(PIC)
+CXXAR := -g -Og -std=c++14 -pedantic -Wall -Wextra -pipe $(INCCXXAR) $(PIC)
 
 OUTDIR := out/
 BLDDIR := out/build/
